Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1805,6 +1805,16 @@ bool RenderFlexibleBox::resolveFlexibleLengths(FlexSign flexSign, FlexItems& fle
     return !totalViolation;
 }
 
+inline ContentPosition resolveLeftRightAlignment(ContentPosition position, const RenderStyle& style, bool isReversed)
```

Comment:
I think it would be nice if the function name stated what it was resolving the passed in alignment values to

Reply:
I think `resolveLeftRightAlignmentToStartEnd` would be more confusing than leaving it blank, actually, because then it reads as if maybe we're mapping left to start and right to end.

The contract here is that left/right are mapping to something other than left/right, so I think it's best left as-is?